server/utils/timezones.go

```diff
@@ -0,0 +1,608 @@
+package utils
+
+var windowsToIanaMap = map[string]string{}
+
+func ConvertWindowsTimezoneToIANA(winTZ string) string {
+	if len(windowsToIanaMap) == 0 {
+		populateConversionMap()
+	}
+
+	tz, ok := windowsToIanaMap[winTZ]
+	if ok {
+		return tz
+	}
+
+	return ""
+}
+
+func populateConversionMap() {
+	for _, entry := range allTimezones {
+		windowsToIanaMap[entry[1]] = entry[0]
+	}
+}
+
+var allTimezones = [][]string{
+	{"Africa/Abidjan", "Greenwich Standard Time", "+00:00"},
```

Comment:
This method of timezone name conversion is not ideal, but I have not found any conversions between Go-compliant and MS Graph-compliant datetimes available to use.

Source of the data in the array:
https://techsupport.osisoft.com/Documentation/PI-Web-API/help/topics/timezones/iana.html

Reply:
1/5 file a HW ticket to have a `make tz` that would pull the list from the link ^^ and generate the go map. This list is not as constant as one would like. Bonus: have a hardcoded default version, and pull the current one dynamically on startup.